src/main/java/uk/gov/verifiablelog/VerifiableLog.java

```diff
@@ -41,14 +41,24 @@ public VerifiableLog(MessageDigest messageDigest, MerkleLeafStore merkleLeafStor
     }
 
     /**
-     * The root hash of the Verifiable Log created from all current leaf values ordered by index.
+     * Gets the current root hash of the Verifiable Log created from all current leaf values ordered by index.
      * @return The Merkle Tree root hash
      */
-    public byte[] currentRoot() {
+    public byte[] getCurrentRootHash() {
         return subtreeHash(0, merkleLeafStore.totalLeaves());
     }
 
     /**
+     * Gets the specific root hash of the Verifiable Log from the given start index and for the given tree size.
+     * @param startIndex The zero-based index of the first leaf in the subtree
+     * @param treeSize The number of leaves in the Verifiable Log which should be included in the root hash
+     * @return The Merkle Tree root hash
+     */
+    public byte[] getSpecificRootHash(int startIndex, int treeSize) {
+        return subtreeHash(startIndex, treeSize);
+    }
+
```

Comment:
This will now be changed to be a zero start index; for our requirements the start index will always be zero, but that might not necessarily be the case for anyone who consumes this library.